core/src/test/java/org/elasticsearch/index/IndexingSlowLogTests.java

```diff
@@ -62,6 +63,16 @@ public void testSlowLogParsedDocumentPrinterSourceToLog() throws IOException {
         p = new SlowLogParsedDocumentPrinter(index, pd, 10, true, 3);
         assertThat(p.toString(), containsString("source[{\"f]"));
         assertThat(p.toString(), startsWith("[foo/123] took"));
+
+        // Throwing a error if source cannot be converted
+        source = new BytesArray("invalid");
+        pd = new ParsedDocument(new NumericDocValuesField("version", 1), SeqNoFieldMapper.SequenceIDFields.emptySeqID(), "id",
+            "test", null, null, source, XContentType.JSON, null);
+        p = new SlowLogParsedDocumentPrinter(index, pd, 10, true, 3);
+
+        assertThat(p.toString(), containsString("_failed_to_convert_[Unrecognized token 'invalid':"
```

Comment:
Sorry, what I meant by the previous request was to do an assertion on the whole error string (e.g. wie assertEquals), unless there are any reasons preventing this.

Reply:
Right, I didn't see your comment above. Makes sense to me now, sorry for the noise.